plugin.py

```diff
@@ -65,10 +65,39 @@ def __init__(self, user, content):
         self.likes = randrange(self.retweets, 2 * self.retweets) # nosec
         self.replies = randrange(self.retweets) # nosec
 
+    # https://stackoverflow.com/a/19213360/1617563
```

Comment:
Would you mind dropping this line?

Reply:
sure thing, I rebased my master on yours (to enable fast forward merge) and squashed this line. I left the formatting changes as a separate commit since `black` reformatted a couple of other lines as well.